API/PROPERTIES

```diff
@@ -33,6 +33,11 @@
     Until we find the answer to those questions (for each and every alias)
     we will continue to regard aliases as noise and thus remove/deprecate them.
 
+    CONSEQUENCE
+
+    "Constr" module does not appear in the API because whatever it provides
+    is also provided by "Term" (which is already present in the API).
+
```

Comment:
Indeed I think policy has to be updated; API should provide compatibility aliases and all the aliases that are present in the base coq interfaces. If an alias is to be removed, it should be first done in the base Coq, then allow some time for the plugin writers to adapt to it.